lib/ai/tools/update-kb-entry.ts

```
@@ -0,0 +1,70 @@
+
+      const entry = result.data;
+
+      return {
+        success: true,
+        message: `Knowledge base entry "${entry.title}" updated successfully.`,
+        entry: {
+          id: entry.id,
+          title: entry.title,
+          summary: entry.summary,
+          content: entry.content.length > 200 
+            ? entry.content.substring(0, 200) + '...' 
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)